test/fixtures/todo/json5/deserialization/number.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'binary number starting with 0b',
+    value: 42,
+    serialized: '0b101010'
```

Comment:
This shouldn't be in a folder named "json5" because JSON5 doesn't allow these binary number literals.